### ISSUE TITLE
As an Industry Reporter I want to see a 'More' column in my operations table in the reporting dashboard so that I can do additional actions for reporting

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#223]
#### Description:

We need to let operators take some additional actions from the Reporting Dashboard page. This includes navigating to the"edit report" "report history" page, the "download .csv" page (TBC) and "download LFO report" page (also TBC) 

[Wireframe here](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7789%3A884&mode=design&t=cSGQYznXhtd4rwZd-1)

![Image](https://github.com/bcgov/cas-reporting/assets/99216375/e4a1468e-dcac-4414-b8ac-53375873793d)

#### Acceptance Criteria:

Given that I am an external operator
And I view the operation list
Then the last column contains a 3 dot button

Given that I am an external operator
When I view the operation list
And I click on the 3 dot button
Then a menu opens with the above mentioned items

Given that I am an external operator
When I view the operation list
And I click on a menu item
Then nothing happens

#### Development Checklist:

- [x] A contextual menu (MUI `Menu` type)
- [x] Clicking on the 3 dots opens the menu
- [x] Menu contains placeholder options which do nothing when clicked

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements (TBC)
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Review with UX Designer
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By

#217 

#### Notes:

-

**Comment by suhafa** [comment]
**Design Review:**
- Flagging that we don't have 'Download LFO' anymore, and devs clarified that it is a placeholder text that can be removed easily
- @BallardRobinett demoed this during stand, looks good overall.
